src/CloudNimble.DotNetDocs.Sdk/CloudNimble.DotNetDocs.Sdk.csproj

```diff
@@ -19,65 +21,55 @@
         
         <!-- Suppress dependencies when packing -->
         <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddPerTFMDocsTasksToPackage</TargetsForTfmSpecificContentInPackage>
```

Comment:
This is the magic - the packaging logic you had before was reconstructing part of what NuGet already gives you. When you pack a nuget package, they give you some [extension points](https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#advanced-extension-points-to-create-customized-package) for easily and correctly adding TFM-specific assets to the package. Your Task dlls and their dependencies are an example of such TFM-specific assets, so we leverage this functionality now.